test/api/projects/instance-get/003-get-ok/expected-response.json

```diff
@@ -1,7 +1,7 @@
 {
   "versionDetail": {
     "components": [],
-    "longDescription": "",
+    "longDescription": "A long **description**.\n",
```

Comment:
A v tomto commitu přidávám `description.md` do repozitáře abych ověřil že se u requestů na GET instance aplikované šablony a GET detailu verze šablony opravdu vypisuje. (Na těchto dvou nás to extra zajímá kvůli UI. Ale i API requesty z předchozího commitu sdílí ten samý kód, tak by to mělo fungovat i u nich.)